src/client.ts

```diff
@@ -1,118 +1,87 @@
 import * as VError from 'verror'
-import { Cache } from './cache'
-import { ClientLowLevel } from './clientLowLevel'
+import axios, { AxiosStatic } from 'axios'
 import Debug from './debug'
-const debug = Debug.extend('high')
+
+const debug = Debug.extend('client')
 
 interface Config {
-  cache?: Cache
-  retries: number
-  lowLevelClient: ClientLowLevel
+  uri: any
+  retries: any
 }
```

Comment:
would be good if it also allows optionally to provide axios instance - might be helpful for providing mocks or a pre-configured axios instance.